tutorial03_traceable_layers/app/src/main/jni/Application.mk

```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 The Android Open Source Project
+# Copyright (C) 2015 Valve Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+APP_ABI := armeabi-v7a arm64-v8a x86 x86_64 mips mips64
+APP_PLATFORM := android-22
+APP_STL := gnustl_static
+APP_CPPFLAGS += -std=c++11 -DVK_PROTOTYPES -Wall -Werror -Wno-unused-variable -Wno-unused-function -Wno-unused-const-variable -DUSE_DEBUG_EXTENTIONS -DVK_USE_PLATFORM_ANDROID_KHR
```

Comment:
This should probably move to Android.mk. Otherwise these will apply to any modules being imported. That was an issue for me when trying to include validation layers and the compile options don't match what they expected.